src/App.tsx

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handlerKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handlerKeyUp);
+  }
+
+  handlerKeyUp = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: event.key,
```

Comment:
let's use `event.code` to check if the `space` is pressed and write `[space]` if so
<img width="191" alt="image" src="https://user-images.githubusercontent.com/55802630/182879782-cef617f4-4d05-4d40-a41f-fb149fac8a14.png">